public/js/main.js

```diff
@@ -1,5 +1,5 @@
 const changeStatusBtn = document.querySelectorAll('.inactiveShift')
-const deleteShiftBtn = document.querySelectorAll('.deleteShift')
+const removeShiftBtn = document.querySelectorAll('.removeShift')

```

Comment:
Nitpick, but I would have named this (and the other vars in this section) `...Btns`. They are arrays of buttons, not single buttons. Variable names are hard. We are constantly trying to make them as descriptive as possible without making them way too long. Plural vs singular is usually a very good one to focus on though.